packaged_releases/README.md

```diff
@@ -3,67 +3,32 @@ Creating a Packaged Release
+Throughout this document, `{VERSION}` refers to a semantic version for this packaged release (e.g. `0.2.3`).
 
 
-1 - Assemble module source code
--------------------------------
-
-To assemble the source code, visit [Releases](https://github.com/Azure/azure-cli/releases) and download the source code for each component (and version) that should be part of the packaged release.  
+1 - Create release archive
+--------------------------
 
-Use the downloaded source code to create a directory with the structure outlined below.
+We create a `.tar.gz` containing the source code for the packaged release.  
+This archive will be used as the basis for the Docker, Debian and Homebrew builds as they build from this source.
 
-Make a directory with name `azure-cli_packaged_{VERSION}`:
+Clone the repo afresh:  
 ```
-$ mkdir azure-cli_packaged_{VERSION}
+$ git clone https://github.com/azure/azure-cli
 ```
 
-**IMPORTANT** - The builds expect a certain folder structure.  
-Expected folder structure inside of `azure-cli_packaged_{VERSION}`:
+Run the script to create the release archive from the 'scripts' folder at the repo root:  
 ```
-.
-|-- az.completion
-|-- src
-|   |-- azure-cli
-|       |-- setup.py
-|       `-- etc...
-|   |-- azure-cli-core
-|       |-- setup.py
-|       `-- etc...
-|   |-- azure-cli-nspkg
-|       |-- setup.py
-|       `-- etc...
-|   |-- command_modules
-|       |-- <MODULE_NAME>
-|           |-- setup.py
-|           `-- etc...
+$ cd scripts
+$ python -m automation.release.packaged --version {VERSION} --components azure-cli=VERSION ...
 ```
 
-(A script may be available in the future to make this step more straightforward.)
-
-Notes:  
-- Only the packages that will be in the CLI should be included here; leave out 'optional' components unless there's a specific reason to include any extra components.
-- Make sure the versions of components don't include the `+dev` suffix. Remove these if this is the case.
-
-APPLY ANY PATCHES:  
-Modify the file in question in the directory created from  (You can use the `patch_*` files in `patches` subdirectory for this).  
-
-
-2 - Create release archive
---------------------------
-
-We create a `.tar.gz` containing the source code for the packaged release.  
-This archive will be used as the basis for the Docker, Debian and Homebrew builds as they build from this source.
-
-The archive should have the following name `azure-cli_packaged_{VERSION}.tar.gz`.
-
-Archive the assembled source code from above:
+A full example:  
 ```
-$ tar -cvzf azure-cli_packaged_{VERSION}.tar.gz azure-cli_packaged_{VERSION}/
+$ python -m automation.release.packaged --version 0.2.3 --components azure-cli=2.0.1 acs=2.0.1 appservice=0.1.1b6 batch=0.1.1b5 cloud=2.0.0 component=2.0.0 configure=2.0.1 container=0.1.1b4 core=2.0.1 documentdb=0.1.1b2 feedback=2.0.0 find=0.0.1b1 iot=0.1.1b3 keyvault=0.1.1b6 network=2.0.1 nspkg=2.0.0 profile=2.0.1 redis=0.1.1b3 resource=2.0.1 role=2.0.0 sql=0.1.1b6 storage=2.0.1 vm=2.0.1
```

Comment:
Make the command accept a response file instead of typing every module and their versions on the same line. It is too easy to make a mistake in this way.